modules/nw-metallb-configuring-frr-8ks.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-metallb-configuring-frr-k8s_{context}"]
+= Configuring integration of MetalLB and FRR-K8s
+
+.Prerequisites
+
+* You have a cluster installed on bare-metal hardware.
+* You have installed the OpenShift CLI (`oc`).
+* You have logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Set the `bgpBackend` field of the `MetalLB` CR to `frr-k8s` as in the following example:
+
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: MetalLB
+metadata:
+  name: metallb
+  namespace: metallb-system
+spec:
+  bgpBackend: frr-k8s
+----
+
+= Extending FRR
```

Comment:
I'd spend a few words here that this is gonna be merged with the FRRConfiguration generated by MetalLB

Reply:
ok. I've added a few sentences her